metadata-ingestion/src/datahub/ingestion/source/aws/aws_common.py

```diff
@@ -92,6 +92,48 @@ def get_session(self) -> Session:
         else:
             return Session(region_name=self.aws_region, profile_name=self.aws_profile)
 
+    def get_credentials(self) -> dict[str, str]:
```

Comment:
This code is fairly duplicative of the `get_session` code above. I wonder if it's worth refactoring both `get_session` and `get_credentials` to use this common `aws` profile details extraction logic.

Reply:
how about now?

Reply:
Makes sense to use the credentials of the session to me.